### PR TITLE
WIP: Create recruitment

### PR DESCRIPTION
**Pull request by Samiksha416**
Adding a recruitment process section

### CODE REVIEW

File: source/process/Recruitment [hunk 1]
```diff
@@ -0,0 +1,137 @@
+Our mission at Mattermost is to be the primary way people communicate within Enterprise and applications. 
+ Our goals are to become the leading enterprise-grade message solution, we’ll achieve this by:
```
Comment (lfbrock) [inline source/process/Recruitment]: Flow for this sentence needs a bit of work, should it be "Our goal is to become" and "messaging solution"? 

I think the intro to the list could be improved as well, ideally it should read like a sentence: 
"We’ll achieve this by #1 private cloud solution by deployments (12-18 months)" doesn't make a lot of sense

File: source/process/Recruitment [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+#1 private cloud solution by deployments (12-18 months)
+#1 private cloud solution by revenue
+#1 solution by deployments (private cloud & SaaS)
+#1 solution by revenue (private cloud & SaaS)
+
+Our Values
+Values are deliberate choices defining our behavior. When facing complexity, uncertainty, or ambiguity (CUA) we determine our point of view and our actions through the lens of our values:
+
+Customer Obsession - We exist to make customers successful. In everything we do, start with customer perspective and work backwards. Earn and keep their trust.
+Ownership - Own the outcomes of your activity. Don’t drop the ball. When we see a vacuum on something important, we jump in–we never say “it’s not my job.”
```
Comment (lfbrock) [inline source/process/Recruitment]: @lindy65 - should we edit "jump in–we never" to have a space around the dash, to be consistent with the beginning of the sentence?

File: source/process/Recruitment [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+High Impact - Figure out what matters most, and focus on those priorities. When deciding to work on low impact or high impact projects, choose high impact.
+Self-Awareness - We understand and seek to understand our strengths and growth opportunities, as individuals and organizations. We are open to critique and share critique constructively and respectfully.
```
Comment (lfbrock) [inline source/process/Recruitment]: Should this be "organizations" or "organization"

File: source/process/Recruitment [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+ 
+Non-Engineering Recruitment Process: 
+
+1. Applicant Review 
+
+Recruiter(s) will review inbound and network referral applications for all assigned requisitions and will archive/screen as is appropriate
+Hiring manager to review shortlist of candidates to further qualify the candidates that should be screened by recruiter
+2. Recruiter Screen 
+
+Conduct requisition specific phone screen to identify skill-sets and experience relevant to Mattermost’s needs, values and compensation expectations.
```
Comment (lfbrock) [inline source/process/Recruitment]: change to "skill sets"

File: source/process/Recruitment [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+
+
+At Mattermost we hire the best of the best to help us attain our mission while portraying our values.  Our recruitment process follows 8-9 steps depending on the role.  
+ 
+Non-Engineering Recruitment Process: 
+
+1. Applicant Review 
+
+Recruiter(s) will review inbound and network referral applications for all assigned requisitions and will archive/screen as is appropriate
```
Comment (lfbrock) [inline source/process/Recruitment]: @lindy65 should we edit this to be a consistent tense/voice? 

ie
"Recruiter(s) review inbound..."
"Hiring manager reviews shortlist..."

File: source/process/Recruitment [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+3. Hiring Manager Screen (Non-engineering)
+
+Schedule 1st Hiring Manager screen
+Review past conversations/notes and resume to prepare for your conversation.
+At the beginning of the interview, introduce yourself, your role and why this position is available
+In conversation with candidate, make sure to “clear sell” the opportunity
+Note all of the following:
+
+PROs / CONs of skill-sets and experience
```
Comment (lfbrock) [inline source/process/Recruitment]: "skill sets"

File: source/process/Recruitment [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+
+Conduct requisition specific phone screen to identify skill-sets and experience relevant to Mattermost’s needs, values and compensation expectations.
+
+3. Hiring Manager Screen (Non-engineering)
+
+Schedule 1st Hiring Manager screen
+Review past conversations/notes and resume to prepare for your conversation.
+At the beginning of the interview, introduce yourself, your role and why this position is available
+In conversation with candidate, make sure to “clear sell” the opportunity
+Note all of the following:
```
Comment (lfbrock) [inline source/process/Recruitment]: is this noting in feedback? 

If so can we make that more clear, so it's obvious these points aren't related to selling the opportunity to the candidate

File: source/process/Recruitment [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+Recruiter(s) will review inbound and network referral applications for all assigned requisitions and will archive/screen as is appropriate
+Hiring manager to review shortlist of candidates to further qualify the candidates that should be screened by recruiter
+2. Recruiter Screen 
+
+Conduct requisition specific phone screen to identify skill-sets and experience relevant to Mattermost’s needs, values and compensation expectations.
+
+3. Hiring Manager Screen (Non-engineering)
+
+Schedule 1st Hiring Manager screen
+Review past conversations/notes and resume to prepare for your conversation.
```
Comment (lfbrock) [inline source/process/Recruitment]: remove period to be consistent with other points

File: source/process/Recruitment [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+At the beginning of the interview, introduce yourself, your role and why this position is available
+In conversation with candidate, make sure to “clear sell” the opportunity
+Note all of the following:
+
+PROs / CONs of skill-sets and experience
+Culture fit evaluation
+List of potential projects they will support
+Potential Title and direct Supervisor
```
Comment (lfbrock) [inline source/process/Recruitment]: remove capitalization

File: source/process/Recruitment [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+Advise on next steps and additional competencies subsequent interviewer(s) should evaluate
+
+4. Support Manager Screen (Non-Engineering)
+
+Review past conversations/notes and resume to prepare for your conversation.
```
Comment (lfbrock) [inline source/process/Recruitment]: remove period

File: source/process/Recruitment [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+At the beginning of the interview, introduce yourself, your role and how this role impacts your department
+ Note all of the following:
+PROs / CONs of skill-sets and experience
```
Comment (lfbrock) [inline source/process/Recruitment]: skill sets

File: source/process/Recruitment [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+Culture fit evaluation
+Who he/she will interact with your department/team
+Advise on next steps
+
+5. Case study project (Optional - to be decided by Hiring Manager)
+
+Depending on the role the Hiring Manager may request a Case Study as part of the recruitment process. You will be asked to prepare a real life example of one of the requirement of the role.
```
Comment (lfbrock) [inline source/process/Recruitment]: "requirements"

File: source/process/Recruitment [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+
+6. 3rd Screen & Case Study Presentation (Optional)
+
+Presentation to Hiring Manager, Support Manager and CEO 
+7. Offer 
+
+Send note to People Ops to create the offer letter. Provide the following information for the Offer letter:
```
Comment (lfbrock) [inline source/process/Recruitment]: capitalization on offer letter is inconsistent

File: source/process/Recruitment [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+Current address (if not on resume in Lever)
+Email address (if not on resume in Lever)
+Salary
+Options (if applicable)
+Bonus/Commissions (if applicable)
+Start Date
+Offer expiry date (usually 3-4 business days)
+Once Offer letter is drafted in Google Doc, Operations will send to CEO for final approval before sending out for electronic signature.
```
Comment (lfbrock) [inline source/process/Recruitment]: match capitalization on offer letter above

File: source/process/Recruitment [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+Salary
+Options (if applicable)
+Bonus/Commissions (if applicable)
+Start Date
+Offer expiry date (usually 3-4 business days)
+Once Offer letter is drafted in Google Doc, Operations will send to CEO for final approval before sending out for electronic signature.  
+
+8. HIRE
+
+Archive candidate in ATS once signed offer received
```
Comment (lfbrock) [inline source/process/Recruitment]: is received

File: source/process/Recruitment [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+Continue to  Welcome email for new hires Workflow
```
Comment (lfbrock) [inline source/process/Recruitment]: extra space, and may want to change to "Welcome Email" or "welcome email"

File: source/process/Recruitment [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+
+
+1. Applicant Review
+
+Recruiter(s) will review inbound and network referral applications for all assigned requisitions and will archive/screen as is appropriate
+Hiring manager to review shortlist of candidates to further qualify the candidates that should be screened by recruiter
+
+2.  Recruiter Screen 
+
+Conduct requisition specific phone screen to identify skill-sets and experience relevant to Mattermost’s needs, values and compensation expectations.
```
Comment (lfbrock) [inline source/process/Recruitment]: skill sets, and if this is a list item remove period

File: source/process/Recruitment [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+
+
+3. Hiring Manager Screen (Engineering)
+Schedule 1st HM screen
+At the beginning of the interview, introduce yourself, your role and why this position is available
+Review past conversations/notes and resume to prepare for your conversation.
```
Comment (lfbrock) [inline source/process/Recruitment]: remove period, + for below list items same comments as above

File: source/process/Recruitment [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+Culture fit evaluation
+List of potential projects they will support
+Potential Title and direct Supervisor
+Advise on next steps and additional competencies subsequent interviewer(s) should evaluate
+4.   Paid Simulation 
+Paperwork sent
+Paperwork completed
+Appropriately nurtured
+Project completed
+Final Interview/follow-up
```
Comment (lfbrock) [inline source/process/Recruitment]: capitalization

File: source/process/Recruitment [hunk 20]
```diff
@@ -0,0 +1,137 @@
...
+
+5. Roundtable (RT) - CEO Meeting
+
+Upon completion of Challenge, HMs and recruiter will discuss final steps.
```
Comment (lfbrock) [inline source/process/Recruitment]: is "Challenge" the "Paid Simulation"

File: source/process/Recruitment [hunk 21]
```diff
@@ -0,0 +1,137 @@
...
+
+6. Reference Checks (RC)
+
+Hiring Manager will send/collect Reference Check using SkillSurvey
```
Comment (lfbrock) [inline source/process/Recruitment]: is capitalization on reference check on purpose?

File: source/process/Recruitment (hunk 1 above)
Comment (Samiksha416) [inline source/process/Recruitment]: It should read: Our goal is to become"

File: source/process/Recruitment (hunk 2 above)
Comment (Samiksha416) [inline source/process/Recruitment]: It should read "When we see a vacuum on something important, we jump-in, we never say “it’s not my job.”  *Note I copy pasted this from the value section on our site, if we change it here it'll have to be changed here too: https://docs.mattermost.com/process/handbook.html#values

File: source/process/Recruitment (hunk 3 above)
Comment (Samiksha416) [inline source/process/Recruitment]: Organization - again let me know if we make the change so I can update here: https://docs.mattermost.com/process/handbook.html#values

File: source/process/Recruitment (hunk 5 above)
Comment (Samiksha416) [inline source/process/Recruitment]: "Recruiter(s) will review"
"Hiring manager will review"

File: source/process/Recruitment (hunk 7 above)
Comment (Samiksha416) [inline source/process/Recruitment]: I'm not sure I follow..but I propose making the change to " Make notes on the following:

File: source/process/Recruitment (hunk 13 above)
Comment (Samiksha416) [inline source/process/Recruitment]: Both should read: Offer Letter

File: source/process/Recruitment (hunk 14 above)
Comment (Samiksha416) [inline source/process/Recruitment]: should be "Offer Letter"

File: source/process/Recruitment (hunk 21 above)
Comment (Samiksha416) [inline source/process/Recruitment]: nope should be 'reference check'

File: source/process/Recruitment (hunk 5 above)
Comment (lindy65) [inline source/process/Recruitment]: I think this means more than one recruiter may review whereas only one hiring manager will review the short list.

File: source/process/Recruitment (hunk 17 above)
Comment (lindy65) [inline source/process/Recruitment]: Only one sentence so I've left the period.

**Comment by lindy65** [comment]
Closing this PR as it has been re-created here: https://github.com/mattermost/docs/pull/1443

**Pull request closed without merging**